### PR TITLE
WF 911926 : Prepare UG document for AutoHideScrollBars visibility support for SfListView control in WinForms

### PR DESCRIPTION
**Pull request by KSaiSivani**
Update the UG documentation for AutoHideScrollBars Visibility Support for SFLiView in Winforms.

### CODE REVIEW

**Comment by Edberg-Vasan** [comment]
rebuild

File: WindowsForms/ListView/ScrollbarCustomization.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+title: Scrollbar customization in Windows Forms ListView Control | Syncfusion
+description: This section explains about the customization of scrollbar Visibility support in Syncfusion Windows Forms (SfListView) control.
+platform: windowsforms
+control: SfListView
+documentation: ug
+---
+
+# Scrollbar customization in Windows Forms ListView Control
+
+This section explains how to customize the scrollbar visibility in the SfListView.
```
Comment (Manivannan-E) [inline WindowsForms/ListView/ScrollbarCustomization.md:12]: @KSaiSivani This section explains how to customize the scrollbar visibility and behavior in the SfListView control.

File: WindowsForms/ListView/ScrollbarCustomization.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## AutoHideScrollBars
+
+SfListView allows to automatically show or hide horizontal and vertical scrollbars using the `AutoHideScrollBars` property. By default, `AutoHideScrollBars` will be set to true. This property manages scrollbar visibility based on content overflow. 
+
```
Comment (Manivannan-E) [inline WindowsForms/ListView/ScrollbarCustomization.md:17]: @KSaiSivani 
The `SfListView` allows you to automatically show or hide horizontal and vertical scrollbars using the `AutoHideScrollBars` property. By default, `AutoHideScrollBars` is set to true. This property manages scrollbar visibility based on content overflow.

Add api reference link for the APIs

File: WindowsForms/ListView/ScrollbarCustomization.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+{% endhighlight %}
+
+{% endtabs %}
+         
+ ![ScrollbarCustomization](ScrollbarCustomization_images/ScrollbarCustomization_img1.png)       
+                             
+## Customizing the Visibility of ScrollBars
+
+### HorizontalScrollBarVisible 
+
```
Comment (Manivannan-E) [inline WindowsForms/ListView/ScrollbarCustomization.md:39]: @KSaiSivani 

The content is mostly clear and well-structured, but I’ve made a few minor grammatical adjustments and improvements for clarity. Here’s a refined version of your UG content:

Scrollbar Customization in Windows Forms ListView Control
This section explains how to customize the scrollbar visibility and behavior in the SfListView control.

AutoHideScrollBars
The SfListView allows you to automatically show or hide horizontal and vertical scrollbars using the AutoHideScrollBars property. By default, AutoHideScrollBars is set to true. This property manages scrollbar visibility based on content overflow.

{% tabs %}

{% highlight c# %}

sfListView1.AutoHideScrollBars = true;

{% endhighlight %}

{% highlight vb %}

sfListView1.AutoHideScrollBars = True

{% endhighlight %}

{% endtabs %}
## Customizing Scrollbar Visibility

HorizontalScrollBarVisible
The HorizontalScrollBarVisible property is used to show or hide the horizontal scrollbar. When AutoHideScrollBars is set to false, the visibility of the horizontal scrollbar will depend on the value of HorizontalScrollBarVisible.

VerticalScrollBarVisible
The VerticalScrollBarVisible property is used to show or hide the vertical scrollbar. When AutoHideScrollBars is set to false, the visibility of the vertical scrollbar will depend on the value of VerticalScrollBarVisible.

File: WindowsForms/ListView/ScrollbarCustomization.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+sfListview1.VerticalScrollBarVisible = True
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![ScrollbarCustomization](ScrollbarCustomization_images/ScrollbarCustomization_img2.png)
+
+## ScrollIncrement
+
```
Comment (Manivannan-E) [inline WindowsForms/ListView/ScrollbarCustomization.md:69]: @KSaiSivani 
## Scroll Increments

### HorizontalScrollIncrement

The `HorizontalScrollIncremen`t property allows you to set the multiplier for mouse wheel scrolling during horizontal scrolling. This property lets users customize the increment value for each scroll step of the mouse wheel.

### VerticalScrollIncrement

The `VerticalScrollIncrement` property allows you to set the multiplier for mouse wheel scrolling during vertical scrolling. Users can customize the increment value for each scroll step of the mouse wheel.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**